### PR TITLE
Feat/intern 1 add seeder master data

### PR DESCRIPTION
**Pull request by nepgya2803**
Add master class seeding methods for those 3 classes: seatclass, airport, aircraft

### CODE REVIEW

File: TicketBooking.Data/DbSeeder/FlightScheDataSeeder.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using TicketBooking.Data.DbContext;
+
+namespace TicketBooking.Data.DbSeeder
+{
+    public class FlightScheDataSeeder : IFlightScheduleDataSeeder
```
Comment (tanngocIT) [inline TicketBooking.Data/DbSeeder/FlightScheDataSeeder.cs]: remove this

File: TicketBookingAPI/Controllers/AircraftController.cs [hunk 2]
```diff
@@ -18,9 +19,14 @@ namespace TicketBookingAPI.Controllers
     public class AircraftController : ControllerBase
     {
         private IAircraftSerivce Aircraftservice { get; }
```
Comment (tanngocIT) [inline TicketBookingAPI/Controllers/AircraftController.cs]: convention

**Review by tanngocIT — APPROVED** [review]

File: TicketBooking.Data/DbSeeder/FlightScheDataSeeder.cs (hunk 1 above)
Comment (nepgya2803) [reply TicketBooking.Data/DbSeeder/FlightScheDataSeeder.cs]: fixed

File: TicketBookingAPI/Controllers/AircraftController.cs (hunk 2 above)
Comment (nepgya2803) [reply TicketBookingAPI/Controllers/AircraftController.cs]: fixed

**Pull request merged**